assets/js/swagger-ui/index.ts

```diff
@@ -0,0 +1,13 @@
+import SwaggerUI from 'swagger-ui'
```

Comment:
To consider this a real option and more than testing it, we could start shaping it like so. Rename the folder to `api-docs` and since this is not an index file, rename it to `apiDocs.ts` or for the time being, may also work to leave it in `assets/js/`